index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+      <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Survey Form</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <form>
+      <h1>Survey Form</h1>
```

Comment:
move the h1 out of the form

Reply:
- [ ] Hello, You did great Using the GitHub inline comment here and also giving a review. :+1: :100: , But I think there are still some improvements you need to make before moving to the next milestone. 
> Instead of giving orders, make suggestions. "I think it would be better to use semantic tags" sounds nicer than "Use semantic tags," doesn't it?
Remember to explain why you request a change - it might be obvious for you but not for someone else.

 - Looking at this point above which was taken from the article on [how to give code review.](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md) I think this your comment here sounds more like an order instead of a suggestion. :+1: :smile: :ringed_planet:  I'll suggest you edit this and rephrase it in a more polite way and also remember to explain why the change is required. :+1: :smile: